articles/libraries/lock/v10/customization.md

```diff
@@ -264,6 +264,34 @@ var options = {
 
 Theme options are grouped in the `theme` property of the `options` object.
 
+#### authButtons {Object}
+
+Allows the customization of the custom oauth2 login buttons. Each custom connection whose button you desire to customize should be listed by name, each with their own set of parameters. The customizable parameters are listed below:
+
+- **displayName** {String}: The name to show instead of the connection name.

```

Comment:
Since this only changes the connection display name not the title of the button, let's change it to something like
`The name to show instead of the connection name when building the button title, e.g. LOGIN WITH MYCONNECTION for login`